.github/workflows/docs-keywords-filter.yml

```diff
@@ -0,0 +1,84 @@
+name: docs-keywords-filter
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+      - release-*
+  pull_request:
+    branches:
+      - master
+      - release-*
+
+# cancel the in-progress workflow when PR is refreshed.
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event_name == 'pull_request' && github.head_ref || github.sha }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+
+jobs:
+  docs-keywords-filter:
+    name: docs-keywords-filter
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Get changed docs
+        id: changed-docs
+        uses: tj-actions/changed-files@v42
+        with:
+          # Avoid using single or double quotes for multiline patterns
+          files: |
+            Documentation/**.md
+            Documentation/**/**.md
+
+      - name: List all changed docs with keywords
+        if: steps.changed-docs.outputs.any_changed == 'true'
+        env:
+          ALL_CHANGED_DOCS: ${{ steps.changed-docs.outputs.all_changed_files }}
+          # Keywords regex for grep
+          GREP_KEYWORD_DICS: 'you\|they'
+          # Error tips
+          TIPS: 'In the docs we try not to use (you,they),please check the file(s) below:'
+        run: |
+
+          echo -e "Changed files: \n ${ALL_CHANGED_DOCS}"
+
+          # Get SHA
+          if [[ "${{ github.event_name }}" == "pull_request" ]];then
+             base_sha=${{ github.event.pull_request.base.sha }}
+             head_sha=${{ github.event.pull_request.head.sha }}
+          else
+            base_sha=${{ github.event.before }}
+            head_sha=${{ github.event.after }}
+          fi
+          echo "Base SHA:${base_sha}"
+          echo "Head SHA:${head_sha}"
+
+          error_msg=''
+          # Grep may return 1 when no added content
+          set +e
+
+          for file in ${ALL_CHANGED_DOCS}; do
+            # Get changed content by 'git diff'
+            # Delete diff summary by sed
+            # Filter added/modified lines , which with a '+' prefix
+            # Match keywords as words and ignore case
+            content_with_keyword="$(git diff ${base_sha} ${head_sha} ${file}|sed '0,/^@/d'|grep '^+'|grep -iw ${GREP_KEYWORD_DICS})"
```

Comment:
can you add a second temporary commit with a document  change having `you` init, to see if this works properly

Reply:
I've added,and the actions result correct
[https://github.com/rook/rook/actions/runs/8325155167/job/22778182972](url)